Library/Homebrew/formula_installer.rb

```diff
@@ -488,6 +494,19 @@ def expand_dependencies(deps)
       end
     end
 
+    if poured_bottle
```

Comment:
Why is this block needed on macOS? How does it change behaviour?

Reply:
`bottle_deps` is empty on macOS. I've added `unless bottle_deps.empty?` to clarify that there is no change in behaviour when `bottle_deps` is empty.